cmd/ctr/run_windows.go

```diff
@@ -9,7 +9,9 @@ import (
 	"github.com/containerd/containerd/containers"
 	"github.com/containerd/containerd/errdefs"
 	"github.com/containerd/containerd/log"
+	"github.com/containerd/containerd/platforms"
 	digest "github.com/opencontainers/go-digest"
+	imageSpecs "github.com/opencontainers/image-spec/specs-go/v1"
```

Comment:
nit, I think we usually have all lowercase: `imagespecs`

Reply:
fixed